plugins/node/opentelemetry-instrumentation-mongodb/src/instrumentation.ts

```diff
@@ -255,7 +256,7 @@ export class MongoDBInstrumentation extends InstrumentationBase {
         const span = instrumentation.tracer.startSpan(`mongodb.${type}`, {
           kind: SpanKind.CLIENT,
         });
-        instrumentation._populateV3Attributes(span, ns, server, cmd);
+        instrumentation._populateV3Attributes(span, ns, server, cmd, type);
```

Comment:
if `commandType === MongodbCommandType.UNKNOWN` maybe it's better to leave this attribute empty instead of writing `command`